packages/cdk8s/test/app.test.ts

```diff
@@ -30,6 +31,58 @@ test('app with two charts', () => {
   ]);
 });
 
+test('app with charts directly dependant', () => {
+
+  // GIVEN
+  const app = Testing.app();
+
+  // WHEN
+  const chart1 = new Chart(app, 'chart1');
+  const chart2 = new Chart(app, 'chart2');
+  const chart3 = new Chart(app, 'chart3');
+
+  Node.of(chart1).addDependency(chart2);
+  Node.of(chart2).addDependency(chart3);
+
+  app.synth();
+
+  // THEN
+  expect(fs.readdirSync(app.outdir)).toEqual([
```

Comment:
Add a test for when only part of the charts have dependencies between them

Reply:
👍